tutorials/notebooks/vo/conesearch.ipynb

```diff
@@ -338,7 +338,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "my_favorite_result = all_results['http://vizier.u-strasbg.fr/viz-bin/conesearch/I/220/out?']\n",
+    "i220keys = [k for k in all_results if 'I/220' in k] #pick out the first one with \"I/220\" in it\n",
```

Comment:
Total nitpick here, but doesn't PEP8 dictate that inline comments should start with `#` + space?

Reply:
While we are nitpicking, it is TWO spaces between code and `#`, and once space after `#`. :wink:

Reply:
I love me my nitpicks/pedantry, so will do, an then merge 😉